test/sdk/go/sdk-client-test.go

```diff
@@ -105,6 +105,9 @@ func main() {
 		testLists(sdk)
 	}
 
+	// Delay before shutdown to prevent Game Servers from churning too quickly on a running cluster
+	time.Sleep(8 * time.Second)
```

Comment:
How is this value decided? Just curious whether make it configurable can make it easy to use.

Reply:
From manually testing different values. 5 seconds led to more unhealthy game servers, and same for 10 seconds. It would be nice to have it configurable, but that might be hard to do since the `sdk-client-test` binary is built at a release version (1.38, 1.39, etc.), and not at run time.